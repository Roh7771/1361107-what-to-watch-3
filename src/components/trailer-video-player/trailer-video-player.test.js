import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./trailer-video-player";

const mock = {
  isPlaying: true,
  videoSrc: `somePath`,
  posterSrc: `somePath`
};

it(`<TrailerVideoPlayer /> should render correctly`, () => {
  const {isPlaying, videoSrc, posterSrc} = mock;
  const tree = renderer
    .create(
        <VideoPlayer
          isPlaying={isPlaying}
          videoSrc={videoSrc}
          posterSrc={posterSrc}
        >
          <video/>
        </VideoPlayer>,
        {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
