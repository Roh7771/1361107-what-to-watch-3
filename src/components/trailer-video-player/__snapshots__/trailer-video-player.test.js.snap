// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrailerVideoPlayer /> should render correctly 1`] = `<video />`;
